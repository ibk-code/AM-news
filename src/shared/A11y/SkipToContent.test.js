import { shallow } from 'enzyme';
import React from 'react';
import SkipToContent from './SkipToContent';

describe('SkipToContent Component test', () => {
  it('expect to render SkipToContent  component', () => {
    expect(shallow(<SkipToContent content="main" />)).toMatchSnapshot();
  });
});
