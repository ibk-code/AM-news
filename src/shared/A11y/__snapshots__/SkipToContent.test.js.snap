// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkipToContent Component test expect to render SkipToContent  component 1`] = `
<a
  className="skip"
  href="#main"
>
  Skip to Content
</a>
`;
